From 11ff3fd1d39d87c1667c414b57f5f1480fe4c3dc Mon Sep 17 00:00:00 2001
From: Jason Smith <jsmith@habana.ai>
Date: Thu, 29 May 2025 08:51:11 -0400
Subject: [PATCH] [SW-230673] pmc: enable pdr auto-gen for agent

Update auto-gen script to also generate the pdr database
for the agent firmware.

Change-Id: I8a3a17597f5d4cb593e0b06117d3a2638bd8798c
Signed-off-by: Jason Smith <jsmith@habana.ai>
---

diff --git a/scripts/g3-gen-all.sh b/scripts/g3-gen-all.sh
index d88cd9b..f8ccb83 100755
--- a/scripts/g3-gen-all.sh
+++ b/scripts/g3-gen-all.sh
@@ -123,7 +123,11 @@
 PREBOOT_INC="${PREBOOT_ROOT}/arch/gaudi3/common/ras/pldm"
 PREBOOT_SRC="${PREBOOT_ROOT}/arch/gaudi3/common/ras/pldm"
 
-for app in MGMT PREBOOT
+AGENT_ROOT="${EMBEDDED_ROOT}/zephyr-plugins"
+AGENT_INC="${AGENT_ROOT}/common/include/gaudi3"
+AGENT_SRC="${AGENT_ROOT}/common/src/gaudi3/ras/pldm"
+
+for app in MGMT PREBOOT AGENT
 do
 	banner "Generating files for ${app}"
 
